dropwizard-rabbitmq/src/main/java/smartthings/dw/rabbitmq/RabbitMQConfiguration.java

```diff
@@ -34,6 +36,8 @@ public int getShutdownTimeout() {
 
     private boolean useNio = false;
 
+    private int requestedHeartBeatTimeoutInSecs = DEFAULT_HEARTBEAT;
```

Comment:
this we are using same default as rabbitmq client. no harm is setting this value as far as i know. We could make this field nullable and prevent from setting if null but i am not inclined to tat idea